src/typedefs.js

```diff
@@ -226,8 +226,8 @@ const React = require('react');
  *
  * @exports documentTableProps
  * @typedef {object} documentTableProps
- * @property {(string) => void} handlePermissions
- * - Function that sets the correct File Name for the SetAclPermissions Modal, and opens it.
+ * @property {(string) => void} handleModal
```

Comment:
I've just check the handler function for handleModal. It looks likes in two parameters, `type` and `fileName`. I believe `type` is also a string(?). If that's the case, the type for handleModal should be updated to be `(type: string, fileName: string) => void` similar to how it's like in TypeScript. See clip below:

https://github.com/codeforpdx/PASS/assets/14917816/9ea0310a-18b1-4b5f-a131-f21c86eacb4f